sgv2-filter.user.js

```diff
@@ -470,6 +508,78 @@ function createFilterUiFilterOptionsRow() {
   flexGrowLeftDiv.appendChild(minLevelToDisplayInput);
   flexGrowLeftDiv.appendChild(levelDashSpan);
   flexGrowLeftDiv.appendChild(maxLevelToDisplayInput);
+    
+  // The "minimal points to display" number input
+  var minPointsToDisplayInput = document.createElement("input");
+  minPointsToDisplayInput.setAttribute("type", "number");
+  minPointsToDisplayInput.setAttribute("maxLength", "3");
+  minPointsToDisplayInput.style.width = "62px";
+  minPointsToDisplayInput.value = minPointsToDisplay;
+  minPointsToDisplayInput.onchange = function() {
+    // Filter out invalid values
+    var minPointsToDisplayInputValue = parseInt(minPointsToDisplayInput.value);
+    if (minPointsToDisplayInputValue < 0 || minPointsToDisplayInputValue > 150 || minPointsToDisplayInputValue > maxPointsToDisplay) {
+      minPointsToDisplayInput.value = minPointsToDisplay;
+    } else if (minPointsToDisplay != minPointsToDisplayInputValue) {
+      // If the value changed, save it and update the UI
+      GM_setValue(KEY_MIN_POINTS_TO_DISPLAY, minPointsToDisplayInputValue);
+      minPointsToDisplay = minPointsToDisplayInputValue;
+      updateFilterCaption();
```

Comment:
You should also add the points to the caption (so the updateFilterCaption call actually has any effect), I'd suggest the caption to look like "Giveaway Filter (Level 0-10, Max 200 entries, X-Y points, Group GAs excluded)".